source/nodejs/adaptivecards/src/scss/adaptivecards-base.scss

```diff
@@ -305,32 +305,24 @@ $font-family: 'Segoe UI', sans-serif;
             }
         }
 
-        &.ac-textInput-readWriteStyle,
-        &.ac-numberInput-readWriteStyle,
-        &.ac-timeInput-readWriteStyle,
-        &.ac-dateInput-readWriteStyle,
-        &.ac-choiceSetInput-compact-readWriteStyle {
-            border: $border-readWrite-style-onrender;
-            background-color: $background-color-readWrite-style-onrender;
+        &.ac-inputStyle-revealOnHover {
+            border: $transparent-border;
+            background-color: $transparent-background;
             &:hover {
-                border: $border-readWrite-style-onhover;
+                border: $revealOnHover-active-border;
             }
         }
 
-        &.ac-choiceSetInput-compact-readWriteStyle {
+        &.ac-choiceSetInput-inputStyle-revealOnHover {
             appearance: none;
         }
     }
 
-    .ac-adaptiveCard:hover .ac-textInput-readWriteStyle,
-    .ac-adaptiveCard:hover .ac-numberInput-readWriteStyle,
-    .ac-adaptiveCard:hover .ac-timeInput-readWriteStyle,
-    .ac-adaptiveCard:hover .ac-dateInput-readWriteStyle,
-    .ac-adaptiveCard:hover .ac-choiceSetInput-compact-readWriteStyle {
-        border: $border-readWrite-style-oncard-hover;
+    .ac-adaptiveCard:hover .ac-inputStyle-revealOnHover {
```

Comment:
Nit: technically not necessary, but should probably still be `.ac-adaptiveCard:hover .ac-input .ac-inputStyle-revealOnHover`